jobclient/python/util.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) Two Sigma Open Source, LLC
+#
+# Licensed under the Apache license, Version 2.0 (the "License");
+# you may not use this file ecept in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import uuid
+
+from uuid import UUID
+
+
+class FetchableUri:
```

Comment:
What is this used for?

Reply:
It's part of the `Job` object, documentation seems to hint at this being a Mesos-centric thing. Despite that, as it is part of the `Job` schema I feel like it'd be better to have a concrete definition for this instead of hiding it behind a more abstract `dict`. Thoughts?